_tests/integration/manual_config_test.go

```diff
@@ -19,25 +21,25 @@ func TestManualConfig(t *testing.T) {
 	tmpDir, err := pathutil.NormalizedOSTempDirPath("__manual-config__")
```

Comment:
nit: I would use `t.TempDir()`, it's automatically cleaned up after the test

Reply:
That is a great recommendation. I have checked the `pathutil.NormalizedOSTempDirPath` function and it is indeed using io function which explicitly states the folders need to be cleaned up manually. I have changed this everywhere.